builtin/plugins/coin/coin.go

```diff
@@ -79,7 +79,7 @@ func (c *Coin) Meta() (plugin.Meta, error) {
 func (c *Coin) Init(ctx contract.Context, req *InitRequest) error {
 	div := loom.NewBigUIntFromInt(10)
 	div.Exp(div, loom.NewBigUIntFromInt(18), nil)
-	//Checks if Coin Policy Feature is enabled before loading Coin Monetary Supply Policy
+	//Checks if Coin Policy is not nil before loading Coin Monetary Supply Policy
```

Comment:
This comment is completely useless, remove it

Reply:
Done.